src/docpage.js

```diff
@@ -10,13 +10,131 @@ export default class DocPage extends React.Component {
     })
   }
 
+
   render() {
     let doc = (this.state || {}).doc
     if(! doc) return <p>loading...</p>
 
+    let files = doc.data.files && doc.data.files.map((item, index) => {
+      return (
+        <tr key={index}>
+          <td>
+            {item.id ?
+              <a href={`./${item.id}`}>{item.filename}</a> :
+              <span>{item.filename}</span>
+            }
+          </td>
+          <td className="text-muted">{item.content_type}</td>
+          <td>
```

Comment:
Also insert a column to display `item.size_pretty`, to mimic current functionality. Maybe omit it if the item is a folder.